### PR TITLE
updates on DSL2

### PR DESCRIPTION
**Pull request by MaxUlysse**
# nf-core/sarek pull request

Many thanks for contributing to nf-core/sarek!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -266,16 +264,18 @@ include { MARK_DUPLICATES }       from './modules/local/mark_duplicates' addPara
...
 
 /*
 ================================================================================
                         INCLUDE nf-core PIPELINE MODULES
 ================================================================================
 */
 
-include { FASTQC }  from './modules/nf-core/fastqc'
-include { MULTIQC } from './modules/nf-core/multiqc'
+include { TRIM_GALORE }     from './modules/nf-core/software/trim_galore.nf'
```
Comment (drpatelh) [inline main.nf:275]:
```suggestion
include { TRIMGALORE }     from './modules/nf-core/software/trimgalore.nf'
```

File: main.nf [hunk 2]
```diff
@@ -266,16 +264,18 @@ include { MARK_DUPLICATES }       from './modules/local/mark_duplicates' addPara
...
+include { MARK_DUPLICATES } from './modules/nf-core/software/mark_duplicates'
```
Comment (drpatelh) [inline main.nf:276]:
```suggestion
include { PICARD_MARKDUPLICATES } from './modules/nf-core/software/picard_markduplicates'
```

File: modules/local/subworkflow/build_indices.nf [hunk 3]
```diff
@@ -6,17 +6,17 @@
 
 // And then initialize channels based on params or indexes that were just built
 
-include { BUILD_INTERVALS }                 from '../local/build_intervals.nf'
-include { BWAMEM2_INDEX }                   from '../nf-core/bwamem2_index.nf'
-include { CREATE_INTERVALS_BED }            from '../local/create_intervals_bed.nf'
-include { GATK_CREATE_SEQUENCE_DICTIONARY } from '../local/gatk_dict.nf'
+include { BUILD_INTERVALS }                 from '../../local/process/build_intervals.nf'
```
Comment (drpatelh) [inline modules/local/subworkflow/build_indices.nf:9]:
```suggestion
include { BUILD_INTERVALS }                 from '../process/build_intervals.nf'
```

File: main.nf (hunk 2 above)
Comment (MaxUlysse) [reply main.nf:276]: it's actually the gatk one, as picard is in gatk now

**Pull request merged**